deploy/rbac-permissions-operator-config/05-osd-readers.ClusterRole.yaml

```diff
@@ -39,51 +39,377 @@ rules:
   verbs:
   - get
   - list
-### START - view ClusterVersion
+### START - run `oc adm top pod` or `oc adm top node`
 - apiGroups:
-  - config.openshift.io
+  - metrics.k8s.io
   resources:
-  - clusterversions
```

Comment:
Why is clusterversion removed?  Add it back please.

Reply:
Clusterversions will be included in line [162](https://github.com/openshift/managed-cluster-config/pull/531/files#diff-10985ab121a609f067ee061b80a75ec59fd445249b4177e68e2168d3131bce00R162):

```
# access resources like oauth, clusterversion,clusteroperators
- apiGroups:
  - config.openshift.io
  resources:
  - oauths
  - '*'
  verbs:
  - get
  - list
  - watch
```